extensions/gitpod-web/src/experiments.ts

```diff
@@ -0,0 +1,91 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Gitpod. All rights reserved.
+ *--------------------------------------------------------------------------------------------*/
+
+import * as vscode from 'vscode';
+import * as configcat from 'configcat-node';
+import * as configcatcommon from 'configcat-common';
+import * as semver from 'semver';
+import Log from 'gitpod-shared/out/common/logger';
+import { URL } from 'url';
+
+const EXPERTIMENTAL_SETTINGS = [
+	'gitpod.experimental.portsView.enabled',
+];
+
+export class ExperimentalSettings {
+	private configcatClient: configcatcommon.IConfigCatClient;
+	private extensionVersion: semver.SemVer;
+
+	constructor(key: string, extensionVersion: string, private logger: Log, gitpodHost: string) {
+		this.configcatClient = configcat.createClientWithLazyLoad(key, {
+			baseUrl: new URL('/configcat', gitpodHost).href,
```

Comment:
From sources it should be gitpod-staging.com, only then we provision to production it should be like that. You can use VSCODE_DEV env var. there should be a client already somewhere with this check.

Reply:
updated